src/Bridges/ApplicationDI/Helpers.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (http://nette.org)
+ * Copyright (c) 2004 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Nette\Bridges\ApplicationDI;
+
+use Nette;
+
+
+/**
+ * Application extension helper.
+ *
+ * @internal
+ */
+class Helpers
+{
+
+	/**
+	 * @return Nette\Application\IPresenter
+	 */
+	public static function createPresenter($class, $container, $touchToRebuild, $invalidLinkMode)
```

Comment:
Typehint for `$container`?

What does `$touchToRebuild` mean? What should be the value? A comment would be nice.